.pipelines/e2e-tests-template.yaml

```diff
@@ -48,11 +48,33 @@ jobs:
           parameters:
             identitySubscriptionID: $(SUBSCRIPTION_ID)
 
+        - script: |
+            export REGISTRY="${REGISTRY:-$(REGISTRY_NAME).azurecr.io/k8s/csi/secrets-store}"
+
+            #This gets latest released chart(tgz) version from charts directory
+            export HELM_CHART_DIR=$(ls -h charts/*.tgz | sort --version-sort --field-separator=- --key=6 | tail -n 1)
+
+            make e2e-test
+          displayName: "Run e2e tests with Released Version"
+          env:
+            AZURE_CLIENT_ID: $(AZURE_CLIENT_ID)
+            AZURE_CLIENT_SECRET: $(AZURE_CLIENT_SECRET)
+            KEY_NAME: $(KEY_NAME)
+            KEY_VERSION: $(KEY_VERSION)
+            KEYVAULT_NAME: $(KEYVAULT_NAME)
+            RESOURCE_GROUP: $(RESOURCE_GROUP)
+            SECRET_NAME: $(SECRET_NAME)
+            SUBSCRIPTION_ID: $(SUBSCRIPTION_ID)
+            TENANT_ID: $(TENANT_ID)
+            IS_UPGRADE_TEST: ${{ parameters.testClusterUpgrade }}
+            ${{ if eq(osType, 'windows') }}:
+              TEST_WINDOWS: true
```

Comment:
Let's set `IMAGE_VERSION` to empty here so the default image version from config is used

https://github.com/Azure/secrets-store-csi-driver-provider-azure/blob/5d1596aeb8d15aad8fb3c81826f978ecca2aa4c6/test/e2e/framework/config.go#L18

This way we will be testing the release image.

Reply:
done